Scatterpolar.md (#5187)
Codecademy/docs: content/plotly/concepts/graph-objects/terms/scatterpolar/scatterpolar.md

---
Title: '.Scatterpolar()'
Description: 'Creates a scatter plot on polar axes.'
Subjects:
  - 'Data Science'
  - 'Data Visualization'
Tags:
  - 'Plotly'
  - 'Graphs'
  - 'Data'
  - 'Values'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---

> The **`.Scatterpolar()`** function in Plotly is used to create scatter plots on polar axes. Scatter plots are used to visualize data points in a two-dimensional space, and polar axes are circular axes that are often used to represent periodic data. 

Reviewer: ```suggestion
The **`.Scatterpolar()`** function in Plotly creates scatter plots on polar axes. Scatter plots are used to visualize data points in a two-dimensional space, and polar axes are circular axes often used to represent periodic data. 
```
Edited the sentence for more clarity and understanding.

Author: I personally love this clarification. Thanks much.

## Syntax

```pseudo
plotly.graph_objects.Scatterpolar(x=None, y=None, ...)
```           

> - `x`: A list of values to be used as the x-coordinates of the data points.  
> - `y`: A list of values to be used as the y-coordinates of the data points.  

Reviewer: ```suggestion
- `x`: A list of values to be used as the x-coordinates of the data points.  
- `y`: A list of values to be used as the y-coordinates of the data points.  
```
Here we should use `r` and `theta` for `plotly.graph_objects.Scatterpolar` because here we are dealing with polar coordinates, not the Cartesian coordinates, so here using z and y would be incorrect. So you can make the correction likewise -
r: It is a list of values representing the radial coordinates of the data points.
theta: It is a list of values representing the angular coordinates of the data points.

Author: duly noted with thanks

> > Note: The ellipsis in the syntax (...) indicates that there are additional optional parameters beyond those listed here to customize the scatter plot on polar axes.   

Reviewer: ```suggestion
> **Note:** The ellipsis in the syntax (...) indicates that there are additional optional parameters beyond those listed here to customize the scatter plot on polar axes.   
```
```suggestion
> **Note:** The ellipsis in the syntax (...) indicates that there are additional optional parameters beyond those listed here to customize the scatter plot on polar axes.   
```
The note word must be mentioned in bold when working with Codecademy docs. Also, I think the lines are redundant here so remove one of them for more clarity.
